### PR TITLE
Added flowchart diagrams

### LINKED ISSUE #502: add flowchart.js diagram
This Issue is an sub-issue of #257 

**Describe the solution you'd like**
Re-add flowchart.js diagram type. This can be done via https://github.com/jln-dk/react-flowchart.js
the language for code blocks was `flow`

### PR DESCRIPTION
**Pull request by DerMolly**
### Component/Part
editor

### Description
This PR readds flowchart.js diagrams

### Steps

- [x] added implementation
- [ ] added / updated tests
- [ ] added / updated documentation
- [ ] extended changelog

### Related Issue(s)
closes #502

### CODE REVIEW

**Comment by DerMolly** [comment]
grafik

File: src/components/markdown-renderer/replace-components/flow/flowchart/flowchart.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  useEffect(() => {
+    if(diagramRef.current === null) {
+      return
+    }
+    const parserOutput = parse(code)
+    parserOutput.drawSVG(diagramRef.current, {
+      'line-width': 2,
+      'fill': 'none',
+      'font-size': '16px',
+      'font-family': 'Source Code Pro twemoji, monospace'
```
Comment (mrdrogdrog) [inline src/components/markdown-renderer/replace-components/flow/flowchart/flowchart.tsx:19]:
```suggestion
      'font-family': 'Source Code Pro, twemoji, monospace'
```

File: package.json [hunk 2]
```diff
@@ -41,6 +41,7 @@
         "eslint-plugin-node": "11.1.0",
         "eslint-plugin-promise": "4.2.1",
         "eslint-plugin-standard": "4.0.1",
+        "flowchart.js": "^1.14.0",
```
Comment (mrdrogdrog) [inline package.json:44]: pin

**Review by mrdrogdrog — CHANGES_REQUESTED** [review]

File: src/components/markdown-renderer/replace-components/flow/flowchart/flowchart.tsx (hunk 1 above)
Comment (mrdrogdrog) [reply src/components/markdown-renderer/replace-components/flow/flowchart/flowchart.tsx]: solved

File: package.json (hunk 2 above)
Comment (mrdrogdrog) [reply package.json]: solved

**Review by mrdrogdrog — APPROVED** [review]

File: src/components/markdown-renderer/replace-components/flow/flowchart/flowchart.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+  if (error) {
+    return (
+      <Alert variant={'danger'}>
+        <Trans i18nKey={'renderer.flowchart.invalidSyntax'}/>
+      </Alert>
+    )
+  } else {
+    return <div ref={diagramRef} className={'text-center'}/>
+  }
```
Comment (ErikMichelson) [inline src/components/markdown-renderer/replace-components/flow/flowchart/flowchart.tsx:48]:
```suggestion
  }
  return <div ref={diagramRef} className={'text-center'}/>
```

**Review by ErikMichelson — CHANGES_REQUESTED** [review]

**Review by ErikMichelson — APPROVED** [review]

**Pull request merged**